orbit-db-cli/test/commands/create.test.ts

```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@oclif/test'
```

Comment:
looks like this is not correctly linted :eyes: 
```suggestion
import { expect, test } from '@oclif/test'
```

Reply:
fixed !